dataviz/dataviz-in-practice.qmd

```diff
@@ -561,12 +561,9 @@ Another way to achieve this is by stacking the densities on top of each other:
 
 ```{r income-smooth-density-counts-by-region-and-year}
 gapminder |> 
-    filter(year %in% years & country %in% country_list) |>
-  group_by(year) |>
```

Comment:
This group/mutate/ungroup pattern seemed dubious to me; it seems to be a remnant from elsewhere - calculated weight isn't used.  I removed the lines locally and the plot is unchanged.